governance.md

```diff
@@ -70,7 +64,7 @@ The election will proceed according to the following process:
 
 1. After a nominated individual(s) agrees to be a candidate for the Oversight Committee, project Maintainers will vote. The voting period will be open for a minimum of three business days. It will remain open until a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of project Maintainers has voted. Only current Maintainers of active projects are eligible to vote.
```

Comment:
Either strike the antithetical (s) or the "a" before nominated.

Reply:
I dont think you need to do either of what @lhawthorn suggests here. The sentence should essentially read "After a nominated individual or individuals" which is what the parenthetical "s" is for.